### PR TITLE
Support multi episode with vllm_v2.

### PR DESCRIPTION
**Pull request by adoda**

### CODE REVIEW

File: chatlearn/schedule/model_manager.py [hunk 1]
```diff
@@ -190,8 +190,7 @@ def sync_parameters(self, episode_offset=0, requires_grad=None, validate=False):
     def set_func_decorator(self, model):
         if is_decorated(model.name):
             return
-        # decorate async method here will raise cannot serialize coroutine object error
-        call_funcs = model.call_funcs if not isinstance(model, VLLMModuleV2) else []
+        call_funcs = model.call_funcs
```
Comment (haolin-nju) [inline chatlearn/schedule/model_manager.py:193]: Is the `time_summary` correct after changing this line of code? The comment here indicates the previous situation.

**Review by haolin-nju — COMMENTED** [review]
See comments

File: chatlearn/schedule/model_manager.py (hunk 1 above)
Comment (adoda) [reply chatlearn/schedule/model_manager.py:193]: time_summary is ok when we using sync_engine, unrelated to this pr.

**Review by haolin-nju — APPROVED** [review]
LGTM

**Review by charles9304 — APPROVED** [review]

**Pull request merged**